packages/design/src/DataTable/types.ts

```diff
@@ -17,13 +17,25 @@ export type TableProps<T> = {
   showFirst?: (data: T[]) => T;
   className?: string;
   style?: React.CSSProperties;
+  // customSort contains fields that describe the current sort direction,
+  // the field to sort by, and a custom sort function.
+  customSort?: CustomSort;
+  // disableFilter when true means to skip running
+  // any client table filtering supplied by default.
+  // Use case: filtering is done on the caller side e.g. server side.
+  disableFilter?: boolean;
```

Comment:
Is this flag necessary? We already have an `isSearchable` prop on the Table and an `isSortable` for each column

Reply:
it's similar to how the existence of `serverSideProps` was working, i just needed a simple flag to do the same thing